### PR TITLE
Add option to disable battle notifications

### PR DESCRIPTION
**Pull request by vasileknik76**
After our team was voting, some members think notifications in the battle is annoying and unnecessary. This feature add option for people could disable notifications.

Following notifications are affected:
- warrior joined
- warrior retreated
- plan skipped
- warrior voted
- warrior retracted vote

### CODE REVIEW

**Comment by StevenWeathers** [comment]
Great idea!

File: frontend/src/pages/Battle.svelte [hunk 1]
```diff
@@ -109,29 +113,35 @@
                 battle.activePlanId = ''
                 battle.votingLocked = true
                 vote = ''
-                notifications.warning(`Plan skipped.`)
+                if ($warrior.notificationsEnabled) {
+                    notifications.warning($_('pages.battle.planSkipped'))
```
Comment (StevenWeathers) [inline frontend/src/pages/Battle.svelte:117]: nice catch ;)

File: frontend/public/lang/default/de.json [hunk 2]
```diff
@@ -277,6 +280,7 @@
             "warriorLeader": "Chef",
             "finalPoints": "Finale Punkte",
             "points": "Punkte",
+            "planSkipped": "Plan skipped.",
```
Comment (StevenWeathers) [inline frontend/public/lang/default/de.json:283]: ```Plan übersprungen``` if you could please, also remove the period in all cases.

Comment (vasileknik76) [reply frontend/public/lang/default/de.json]: Done

**Pull request merged**